### PR TITLE
[5.2] Added wherePivotIn function

### PR DESCRIPTION
**Pull request by hamedmehryar**
Added wherePivotIn and orWherePivotIn functions

### CODE REVIEW

File: src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php [hunk 1]
```diff
@@ -115,6 +115,22 @@ public function wherePivot($column, $operator = null, $value = null, $boolean =
     }
 
     /**
+     * set a "where in" clause for a pivot table column.
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php]: Please start the sentence with a capital letter.

**Comment by hamedmehryar** [comment]
fixed the docstring.

**Comment by taylorotwell** [comment]
Why does this not prepend the table name to the column like wherePivot does?

**Comment by hamedmehryar** [comment]
Fixed. 
Yes it was missing. I was testing it for my case and I was getting the intended results! :confused: 
That was because my pivot column name was unique among column names of tables present in the query.

**Comment by GrahamCampbell** [comment]
Please rebase and force push to avoid the merge commit.

**Pull request merged**